core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -329,6 +329,17 @@ class H2OContext private(val sparkSession: SparkSession, conf: H2OConf) extends
     basic ++ sparkYarnAppId ++ backend.epilog
   }
 
+  /**
+    * @param path directory where the logs will be downloaded
+    */
+  def downloadH2OLogs(path: String, logFileName: String = "logs.zip"): String = {
+    import sys.process._
+    import java.net.URL
+    import java.io.File
+    new URL(s"http://$localClientIp:$localClientPort/3/Logs/download").#>(new File(path, logFileName)).!!
```

Comment:
Please, improve it:
  - there is already distributed task collecting logs, please use it
   - let user to specify the location in the same way as we expose ModelSerializationSupport trait